trunk/NDHMS/hydro_namelists.json

```diff
@@ -104,6 +108,16 @@
         "nudging_nlist": {}
     },
 
+    "nwm_channel-only_forecasts": {
+        "hydro_nlist": {
+            "lsmout_domain": 0,
+            "output_gw": 0,
+            "output_channelbucket_influx": 2,
+            "reservoir_persistence": true,
```

Comment:
Comma at the end of this line (invalid JSON) prevents compiling the candidate - easy fix!

Reply:
Did we want this off for now?

Reply:
Perhaps so, in the same vein as other PRs where we test the new optional code disabled (Q: does this, when ostensibly disabled, affect anything or have any side effects?), then turn it on and retest.